### ISSUE TITLE
Map freezes as popups are cleared upon dragging map

### ISSUE THREAD

**Issue by notacouch** [amcharts/amcharts4#299]
Relevant code:

```
chart.events.on("mappositionchanged", function(event) {
  if (chart.popups.length) {
    chart.closeAllPopups();
    chart.popups.clear(); // <-- key line
  }
});
```

Pen: https://codepen.io/notacouch/pen/9f0441768c1041e8f7e4d648002f6816

This was working before, don't know when it started freezing.

**Comment by martynasma** [comment]
I believe this is related to #312.

Fixed in dev.

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.49](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.49).

## [4.0.0-beta.49] - 2018-09-05

### Added
- JSON config: added `callback` property, which if it holds a reference to function will call it within the scope of the object being prepared.
- Adapters for `url` and `urlTarget` properties.
- Chart can now be panned or zoomed using mouse wheel. To set what it does use `mouseWheelBehavior` on `XYChart` (see next point)
- [`mouseWheelBehavior`](https://www.amcharts.com/docs/v4/reference/xychart/#mouseWheelBehavior_property) added to `XYChart`. Options: `"zoomX"` (default), `"zoomY"`, `"zoomXY"`, `"panX"`, `"panY"`, `"panXY"`, `"none"`.
- [`mouseWheelBehavior`](https://www.amcharts.com/docs/v4/reference/mapchart/#mouseWheelBehavior_property) added to `MapChart`. Options: `"zoom"` (default), `"none"`.

### Changed
- Second parameter to `Cursor`'s [`triggerMove()`](https://www.amcharts.com/docs/v4/reference/cursor/#triggerMove_method) method was changed from boolean to enumeration of "stickiness" level: `"hard"`, `"soft"`, `"none"` (default).

### Fixed
- Error fixed when `Popup` object was being disposed.
- Over-panning of an `XYChart` did not disappear after mouse button was released outside chart.
- Multiple memory leaks fixed.
- `MapChart` was rounding lat/long coordinates too much which was resulting in pixelated maps for small-area maps.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
